cypress/fixtures/esm/package.json

```
@@ -0,0 +1,5 @@
+{
+  "type": "module",
+  "scripts": { "test": "node test.js" },
+  "dependencies": { "a11y-dialog": "file:../dist/a11y-dialog.esm.js" }
```

```suggestion
  "dependencies": { "a11y-dialog": "file:../../" }
```

Here, and elsewhere, **we want this to be the root.** If we point to the exact file, we’re not testing the `exports` in the root package.json

---

Ah yes, that makes total sense! It’s actually `../../..`, but I totally understand what you mean. I am going to clean up that PR from all the unnecessary changes, and only introduce the tests.

---

All updated. :)

---

🙌🏻